src/components/table/Table.vue

```diff
@@ -192,12 +198,26 @@
             /**
              * When data prop change, update internal value and sort again,
              * do not sort however if it's backend-sort.
+             * If it isn't backend-paginated set new total
              */
             data(value) {
                 this.newData = value
                 if (!this.backendSorting) {
                     this.sort(this.currentSortColumn, true)
                 }
+                if (!this.backendPagination) {
+                    this.newDataTotal = value.length
+                }
+            },
+
+            /**
+             * When Pagination total change, update internal total
+             * only if it's backend-paginated.
+             */
+            total(newTotal) {
+                if (this.backendPagination) {
```

Comment:
Change to 
```javascript
if (!this.backendPagination) return

this.newDataTotal = newTotal
```

It's a good practice to return the function as soon as possible.